packages/paste-core/components/select/src/Select.tsx

```diff
@@ -110,6 +110,7 @@ export const SelectElement = React.forwardRef<HTMLSelectElement, SelectProps>(
         width="100%"
         _disabled={{
           color: variant === "inverse" ? "colorTextInverseWeakest" : "colorTextWeaker",
+          backgroundColor: "transparent",
```

Comment:
do we have any way of testing this on windows? might be safer to set it to the actual desired token for disabled rather than transparent

Reply:
I used browerstack to test it and works for me too